packages/valory/skills/market_manager_abci/bets.py

```diff
@@ -58,6 +68,32 @@ class Bet:
     status: BetStatus = BetStatus.UNPROCESSED
     blacklist_expiration: float = -1
 
+    @classmethod
+    def from_gamma_subgraph(cls, **kwargs: Any) -> "Bet":
+        """Initialize a bet's instance from gamma subgraph's attributes."""
+        outcomes = eval(kwargs["outcomes"])
+        outcome_token_amounts = [0] * len(outcomes)
+        end_date = datetime.strptime(kwargs["endDate"], "%Y-%m-%dT%H:%M:%S.%fZ")
+        opening_timestamp = int(end_date.replace(tzinfo=timezone.utc).timestamp())
+        created_by = kwargs["createdBy"] or NULL_ADDRESS
+        outcome_prices = eval(kwargs["outcomePrices"])
+
+        omen_to_polywrap_mapping = {
+            omen_key: kwargs[polywrap_key]
+            for omen_key, polywrap_key in OMEN_TO_POLYWRAP.items()
+        }
+
+        return cls(
+            collateralToken=USDC,
```

Comment:
TODO: make sure this is not used when making a request to the mech, but xdai or wxdai is used.

Note: we need to interact with both gnosis and polygon chains for Polymarket.

Reply:
Why do you need to interact with both chains?

Reply:
To interact with the mech on Gnosis. Is the mech deployed on Polygon?

Reply:
We will deploy it on Polygon eventually. For testing use it on Gnosis